packages/commands/src/route-tokens-to-user-bank.mjs

```diff
@@ -0,0 +1,249 @@
+import {
+  TOKEN_PROGRAM_ID,
+  createAssociatedTokenAccountInstruction,
+  createTransferCheckedInstruction,
+  getAccount,
+  mintTo
+} from '@solana/spl-token'
+import {
+  Connection,
+  Keypair,
+  PublicKey,
+  Transaction,
+  TransactionInstruction,
+  sendAndConfirmTransaction
+} from '@solana/web3.js'
+import chalk from 'chalk'
+import { Option, program } from 'commander'
+
+import { route } from '@audius/spl'
+
+import { initializeAudiusLibs } from './utils.mjs'
+
+const TOKEN_DECIMALS = {
```

Comment:
do we want to pull these out somewhere?

Reply:
We could. The convention in these command scripts seems to be that they are local testing utilities and as such don't follow super duper strict style guide rules. Given that, it didn't seem like a huge deal to duplicate some code.

Reply:
fair nuf